gcloud-java-examples/src/main/java/com/google/gcloud/examples/Stat.java

```diff
@@ -0,0 +1,104 @@
+package com.google.gcloud.examples;
```

Comment:
This will also affect the README command line input examples (they'll need to have "nio" as well).